assets/css/base/_base.scss

```
@@ -22,3 +24,45 @@ h3 {
 p {
   font-size: 1em;
 }
+
+// Links and Buttons
+a {
+  color: $blue-dark;
+  font-weight: bold;
+  text-decoration: none;
+}
+
+a:hover,
+a:focus {
+  background-color: $orange-focus;
+}
+
+button {
+  background-color: $green-light;
+  border: 2px solid $blue-alt-light;
+  border-radius: 70px;
+  font-size: 24px;
+  font-weight: 400;
+  height: 53px;
+  line-height: 34px;
```

It's a good idea to use a unitless number for a line height so that the line height stays proportional if the user resizes the browser font size. See here: https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#Prefer_unitless_numbers_for_line-height_values

The way you figure out this number is by dividing the line height in pixels by the font size in pixels, so in this case: 34 / 24. Sass can do this for you if you set the line height value as follows:

```suggestion
  line-height: (34 / 24);
```

---

got it :)